shared/middleware/helpers/storage_apps.rb

```diff
@@ -13,10 +13,12 @@ def initialize(storage_id)
   end
 
   def create(value, ip_address)
+    timestamp = Time.now
```

Comment:
In the short term, I think it's ok to leave the old code in place that populates data.created_at with Time.now, since that code is going away. So this should just be (1) change Time.now to DateTime.now on the line above and (2) use DateTime.parse in the migration (possibly not necessary but unsure how to verify this)